src/createRollupConfig.ts

```diff
@@ -108,6 +108,7 @@ export async function createRollupConfig(
           'main',
           opts.target !== 'node' ? 'browser' : undefined,
         ].filter(Boolean) as string[],
+        extensions: ['.mjs', '.js', '.jsx', '.json', '.node'],
```

Comment:
Can you add a comment that says like `// defaults + .jsx`?

Just to be explicit since there doesn't seem to be away to import the default extensions (as there is with Babel)

Reply:
Updated